Themis.xcodeproj/project.pbxproj

```diff
@@ -1851,14 +1851,6 @@
 				version = 4.0.4;
 			};
 		};
-		6564DC022A00EFFC00CF2F89 /* XCRemoteSwiftPackageReference "artemis-ios-core-modules" */ = {
```

Comment:
I thought maybe removing this would fix the failing bamboo build, but it didn't. No idea why it's failing 🤷‍♂️

Reply:
👍 Thanks!